src/app/Http/Controllers/Auth/LoginController.php

```diff
@@ -24,23 +24,30 @@ public function index(): View
     public function auth(LoginRequest $request): RedirectResponse
     {
         $validatedData = $request->validated();
-
         $credentials = [
             'email' => $validatedData['email'],
             'password' => $validatedData['password'],
         ];
 
-        if (auth()->attempt($credentials)) {
-            $request->session()->regenerate();
-            return redirect()->route('profile.show');
+        $user = User::where('email', $validatedData['email'])->first();
+
+        if (!$user) {
+            return redirect()->route('login')->with('error', 'Email does not exist. Please register or check the email address.');
         }
```

Comment:
You can validate the database table column using a form request. Check laravel docs